app/sidekiq/immunization_schedule_reminder_job.rb

```diff
@@ -0,0 +1,13 @@
+class ImmunizationScheduleReminderJob
+  include Sidekiq::Job
+
+  def perform()
+    immunizations = Immunization.where(vaccination_date: Date.today + 1.day)
+    immunizations.each do |immunization|
+      ward = Ward.find(immunization.ward_id)
+      users = ward.user
+      date = immunization.vaccination_date
+      ImmunizationScheduleMailer.reminder_email(users, ward, date).deliver_now
```

Comment:
Why not just:

ImmunizationScheduleMailer.reminder_email(immunization.ward.user, immunization.ward, immunization.vaccination_date).deliver_now OR even better why the reminder_email method cannot just accept an immunization object in params?